todo/serializers/defer_task_serializer.py

```
@@ -0,0 +1,11 @@
+from rest_framework import serializers
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)